src/ranged.cpp

```diff
@@ -865,21 +880,56 @@ std::vector<tripoint> game::target( tripoint &p, const tripoint &low, const trip
     ctxt.register_action("PREV_TARGET");
     if( mode == TARGET_MODE_FIRE ) {
         ctxt.register_action("AIM");
-        ctxt.register_action("AIMED_SHOT");
-        ctxt.register_action("CAREFUL_SHOT");
-        ctxt.register_action("PRECISE_SHOT");
+        ctxt.register_action("SWITCH_AIM");
+        aim_types.push_back( aim_type { "", "", "", false, 0 } ); // dummy aim type for unaimed shots
+        const int th_step = 30;
```

Comment:
Please don't be so parsimonious with your letters, full words make things so much more understandable.